.changeset/good-tips-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+"llamaindex": patch
```

Comment:
@himself65 we saw this issue when using nextjs 15 as a warning in create-llama generated code. you might want to release this with the next minor version
```suggestion
"llamaindex": minor
```

Reply:
could we both suport nextjs 14 and 15?

Reply:
I don't want to bump to 0.9.0

Reply:
@himself65 nextjs config object and webpack config object both don't contain nextjs version, so we better wait to merge this PR once nextjs 15 got more popular - right now it works for nextjs 14 and 15 and for 15, we're just getting a warning.